instructionAPI/src/AMDGPU/gfx908/finalizeOperands.C

```diff
@@ -1,2892 +1,2265 @@
+                insn_in_progress->appendOperand(makeRegisterExpression(amdgpu_gfx908::vmcnt,0,36),true,true,true);
+                if(layout.IDXEN){
+                    insn_in_progress->appendOperand(makeRegisterExpression(amdgpu_gfx908::idxen, 1 ),false,false,false);
+                }
+                if(layout.OFFEN){
+                    insn_in_progress->appendOperand(makeRegisterExpression(amdgpu_gfx908::offen, 1 ),false,false,false);
+                }
+                if(layout.OFFSET){
+                    insn_in_progress->appendOperand(NamedImmediate::makeNamedImmediate(std::string("offset"),Result(u16, layout.OFFSET)),true,false,false);
+                }
+                break;
+                case 65:case 96:case 98:case 99:case 100:case 101:case 102:case 103:case 104:case 105:case 106:case 107:case 108:
+                //BUFFER_ATOMIC_CMPSWAP,BUFFER_ATOMIC_SWAP_X2,BUFFER_ATOMIC_ADD_X2,BUFFER_ATOMIC_SUB_X2,BUFFER_ATOMIC_SMIN_X2,BUFFER_ATOMIC_UMIN_X2,BUFFER_ATOMIC_SMAX_X2,BUFFER_ATOMIC_UMAX_X2,BUFFER_ATOMIC_AND_X2,BUFFER_ATOMIC_OR_X2,BUFFER_ATOMIC_XOR_X2,BUFFER_ATOMIC_INC_X2,BUFFER_ATOMIC_DEC_X2
+                appendOPR_VGPR(layout.VDATA,true,true,2);
+                if(layout.IDXEN || layout.OFFEN){
+                    appendOPR_VGPR(layout.VADDR,true,false);
+                }else if(layout.OFFSET){
+                    insn_in_progress->appendOperand(makeRegisterExpression(amdgpu_gfx908::off, 1 ),false,false,false);
+                }
+                if(layout.IDXEN && layout.OFFEN){
+                    appendOPR_VGPR(layout.VADDR+1,true,false);
+                }
+                appendOPR_SREG(layout.SRSRC,true,false,4);
+                appendOPR_SSRC_NOLIT(layout.SOFFSET,true,false);
+                insn_in_progress->appendOperand(makeRegisterExpression(amdgpu_gfx908::vmcnt,0,36),true,true,true);
+                if(layout.IDXEN){
+                    insn_in_progress->appendOperand(makeRegisterExpression(amdgpu_gfx908::idxen, 1 ),false,false,false);
+                }
+                if(layout.OFFEN){
+                    insn_in_progress->appendOperand(makeRegisterExpression(amdgpu_gfx908::offen, 1 ),false,false,false);
+                }
+                if(layout.OFFSET){
+                    insn_in_progress->appendOperand(NamedImmediate::makeNamedImmediate(std::string("offset"),Result(u16, layout.OFFSET)),true,false,false);
+                }
+                break;
+                case 97:
+                //BUFFER_ATOMIC_CMPSWAP_X2
+                appendOPR_VGPR(layout.VDATA,true,true,4);
+                if(layout.IDXEN || layout.OFFEN){
+                    appendOPR_VGPR(layout.VADDR,true,false);
+                }else if(layout.OFFSET){
+                    insn_in_progress->appendOperand(makeRegisterExpression(amdgpu_gfx908::off, 1 ),false,false,false);
+                }
+                if(layout.IDXEN && layout.OFFEN){
+                    app
```

Comment:
Put each `if` block statement on its own line.  In general only 1 statement per line.

Reply:
@kupsch I'm not sure what you mean in this comment.

Reply:
I highlighted different lines. Don't know what happened. They were in `instructionAPI/src/AMDGPU/gfx908/finalizeOperands.C` lines 1691-1693:

```
                if (vmcnt != 0x3f) {insn_in_progress->appendOperand( makeRegisterExpression(amdgpu_gfx908::vmcnt, 0 ,32),false,true);insn_in_progress->appendOperand( Immediate::makeImmediate(Result(u32,vmcnt)),false,false);}
                if (expcnt != 0x7) {insn_in_progress->appendOperand( makeRegisterExpression(amdgpu_gfx908::expcnt, 0 ,32),false,true);insn_in_progress->appendOperand( Immediate::makeImmediate(Result(u32,expcnt)),false,false);}
                if (lgkmcnt != 0xf) {insn_in_progress->appendOperand( makeRegisterExpression(amdgpu_gfx908::lgkmcnt, 0 ,32),false,true);insn_in_progress->appendOperand( Immediate::makeImmediate(Result(u32,lgkmcnt)),false,false);}
```

Tge lines are indented too little and the `if` block is on the same line, and the block contains multiple statements.

Reply:
I see. Probably because I rebased on master and forced-pushed. 
Not sure what you meant by Tge lines, but let me try to fix this.

Reply:
Tge -> The